Dockerfile-gradle-with-tc

```diff
@@ -0,0 +1,2 @@
+FROM gradle:6.9.0-jdk8 AS gradle-build
+COPY --from=jetbrains/teamcity-server:2021.1.2 /opt/teamcity /teamcity
```

Comment:
is there a reason we need to copy in the teamcity? The build itself doesn't depend on it - gradle pulls are necessary dependencies.
(this probably goes along with the comment further down) - which may then mean this file be removed.

Reply:
Yep, this can go now, was only done so it could get the TC directory. This whole file can go now.